Core/Utilities/DErrno.py

```diff
@@ -175,7 +180,12 @@
                1604 : "EFCERR",
 
                # RMS
-               1700 : "ERMSUKN",
+               1700 : ERMSUKN,
+
+               # Resources and RSS
+               2000 : ERESGEN,
```

Comment:
This is indeed a bug. It should be 

    2000 : 'ERESGEN',

And similar for other errors